### ISSUE TITLE
sqlite3 & drift database support

### ISSUE THREAD

**Issue by egyleader** [serverpod/serverpod#160]
would be nice if have [sqlite3](https://pub.dev/packages/sqlite3) based databases support instead of relying on postgresql and docker  for small projects ,, this will make it possible to integrate orm solutions like [drift](https://pub.dev/packages/drift) for easy creation and minpulation of the database .

**Comment by vlidholt** [comment]
We will probably implement this in the future, as it would make it possible to run Serverpod on devices too.

**Comment by egyleader** [comment]
yes this would make serverpod much more popular as a backend solution for dart and flutter ,, I can help in making this support if you can guide me through the right way to do this  as I am new into building something that scale.

**Comment by runyaga** [comment]
It would be nice to say why this wasn't initially used at start. Pretty sure drift supports postgresql (maybe drift didnt support pgsql when server pod was originally designed?). I would imagine this would be a FAQ for ppl with flutter experience.

**Comment by SandPod** [comment]
As long as the running database supports Postgresql DSL and drivers, Serverpod is compatible. So if drift now supports pgsql it should be possible to use together with Serverpod.
